Resources/doc/versioning.rst

```diff
@@ -0,0 +1,180 @@
+API versioning
+==============
+Why version your API ?
```

Comment:
Thanks and sorry it's the first documentation I wrote, I didn't know all this standards...
I'll try to read the reStructuredText doc soon.